modules/tiles/src/tileset/tileset-3d.ts

```diff
@@ -394,12 +395,15 @@ export default class Tileset3D {
    * @deprecated
    */
   update(viewports: any[] | null = null) {
-    if (!viewports && this.lastUpdatedVieports) {
-      viewports = this.lastUpdatedVieports;
-    } else {
-      this.lastUpdatedVieports = viewports;
-    }
-    this.doUpdate(viewports);
+    // eslint-disable-next-line @typescript-eslint/no-floating-promises
+    this.tilesetInitialization.then(() => {
```

Comment:
Can we use async / await syntax?

Reply:
`update` is synchronous function that is called from @deck.gl.
Now it is `@deprecated`. I'll create a PR in @deck.gl after 3.2 to use `async selectTiles` method instead of `update`